src/controller/cve.controller/index.js

```diff
@@ -249,7 +225,7 @@ router.put('/cve/:id',
         <p>User must belong to an organization with the <b>Secretariat</b> role</p>
         <h2>Expected Behavior</h2>
         <p><b>Secretariat:</b> Updates a CVE record for any organization</p>"
-  #swagger.parameters['id'] = { description: 'The CVE ID for which the record is being updated.' }
+  #swagger.parameters['id'] = { description: 'The CVE ID for the record being updated' }
   #swagger.parameters['$ref'] = [
     '#/components/parameters/apiEntityHeader',
```

Comment:
Response body should match this. I think you just need to change 'created' to 'updated' to get this from the POST response body
{
    "message": "CVE-1999-0005 record was successfully updated.",
    "updated": {
        "dataType": "CVE_RECORD",
        "dataVersion": "5.0",
        "cveMetadata": {
            "cveId": "CVE-1999-0005",
            "assignerOrgId": "ee1fa4c6-8d34-4353-ad97-194c1b986b8b",
            "state": "PUBLISHED",
            "assignerShortName": "exampleCNA",
            "requesterUserId": "adfb94fe-a90a-42a5-8ce4-12aa1f897cde",
            "dateReserved": "2022-04-04T02:28:12.844Z",
            "datePublished": "2022-04-04T02:29:23.283Z"
        },
        "containers": {
            "cna": {
                "x_a": {
                    "bb": "c"
                },
                "affected": [
                    {
                        "product": "p",
                        "vendor": "v",
                        "versions": [
                            {
                                "version": "1.2",
                                "status": "affected"
                            }
                        ]
                    }
                ],
                "descriptions": [
                    {
                        "lang": "en",
                        "value": "v p 1.2 is insecure."
                    }
                ],
                "problemTypes": [
                    {
                        "descriptions": [
                            {
                                "description": "insecurity",
                                "lang": "en"
                            }
                        ]
                    }
                ],
                "providerMetadata": {
                    "orgId": "ee1fa4c6-8d34-4353-ad97-194c1b986b8b",
                    "shortName": "exampleCNA"
                },
                "references": [
                    {
                        "url": "https://newexample.com"
                    }
                ]
            }
        }
    }
}